lisa/tools/hyperv.py

```diff
@@ -53,15 +73,61 @@ def exists_vm(self, name: str) -> bool:
 
         return bool(output.strip() != "")
 
+    def get_vm_disks(self, name: str) -> List[VMDisk]:
+        vm_disks: List[VMDisk] = []
+        output = self.node.tools[PowerShell].run_cmdlet(
+            f"Get-VMHardDiskDrive -VMName {name} ",
+            force_run=True,
+            output_json=True,
+        )
+        # above command returns a list of disks if there are multiple disks.
+        # if there is only one disk, it returns a single disk but not a list.
+        # so convert the output to a list if it is not already a list
+        if not isinstance(output, List):
```

Comment:
```
if not output:
    return []
if not isinstance(output, list):
    output = [output]
```

Reply:
Fixed

Reply:
@LiliDeng  Please let me know if anything more is needed on this?